alacritty_terminal/src/event_loop.rs

```diff
@@ -313,17 +311,12 @@ where
             let mut state = State::default();
             let mut buf = [0u8; READ_BUFFER_SIZE];
 
-            let mut tokens = (0..).map(Into::into);
-
-            let poll_opts = PollOpt::edge() | PollOpt::oneshot();
-
-            let channel_token = tokens.next().unwrap();
-            self.poll.register(&self.rx, channel_token, Ready::readable(), poll_opts).unwrap();
+            let mut tokens = 0..;
```

Comment:
Could we remove this token increments we had before? I guess we can use a static token for the channel, no?